Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/GDSInstructions/GDSInstructionsViewController.swift

```diff
@@ -41,6 +41,13 @@ public class GDSInstructionsViewController: UIViewController {
         primaryButton.isEnabled = true
         primaryButton.isLoading = false
         setBackButtonTitle()
+        
+        if viewModel.rightBarButtonTitle != nil {
```

Comment:
Is there a reason we use this rather than `if let`?

Reply:
I think it achieves a similar result. The init for the bar button item takes an optional string so I don't actually need to unwrap the property; just checking that the property isn't nil. Happy to change it if you'd prefer though?